src/WorkflowServiceProvider.php

```diff
@@ -50,75 +49,57 @@ function ($property = 'marking', PropertyAccessorInterface $propertyAccessor = n
             return new Registry();
         };
 
-        $app['workflow.config'] = [];
```

Comment:
I'll prefer to remain this service parameter for two reasons:
1. it's a sort of "fallback" in case no config was provided by user.
2. one can add workflow configurations from any place after workflow service provider registration:

```php
    <?php
    // some service provider
    $app->extend('workflow.config', function (array $config) {
        $config[] = ... // here comes workflow configuration
    });
```
or simply

```php
    <?php
    // yet another service provider
    $app['workflow.config'][] = ... // here comes another workflow configuration